main.go

```diff
@@ -243,6 +247,18 @@ func visit(url *url.URL) {
 	}
 }
 
+func createBody(body string) (io.ReadCloser, error) {
+	if strings.HasPrefix(body, "@") {
+		filename := body[1:]
+		f, err := os.Open(filename)
+		if err != nil {
+			return nil, fmt.Errorf("failed to open data file %s: %v", filename, err)
+		}
+		return f, nil
+	}
+	return ioutil.NopCloser(strings.NewReader(body)), nil
```

Comment:
That was because I thought at first I needed to be an io.Closer in order to be able to close it later. Once I found out it was handled by `net/http` automatically, this became unnecessary.